### PR TITLE
Add CompareIndicators Script

### PR DESCRIPTION
**Pull request by DeanArbel**
## Status
- [x] Ready

## Related Issues
helps us with: https://github.com/demisto/etc/issues/39700

## Description
Added a script to find the differences between two indicators lists.

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    diff2 = ip_groups_to_ranges(ip_diff2.iter_ipranges())
+    diff1.update(non_ip_set_1.difference(non_ip_set_2))
+    diff2.update(non_ip_set_2.difference(non_ip_set_1))
+    return diff1, diff2
+
+
+def main():
+    args = demisto.args()
+    ioc_list1 = argToList(args.get('ioc_list1', []))
+    ioc_list2 = argToList(args.get('ioc_list2', []))
```
Comment (bakatzir) [inline Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.py:66]: I would consider another arg names. e.g:
`base_list`, `compare_to_list` - your call.
also, for volume purposes and the fact that many customers use the `panosurl` format with hosting files, consider adding the option to parse entryID of a file containing a list. your call if worth the hassle.

File: Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+fromversion: 6.0.0
+script: ''
+type: python
+tags: []
+comment: Find the differences between two indicators lists.
+enabled: true
+args:
+- name: base_list
+  required: true
+  description: Indicators of Compromise list.
```
Comment (bakatzir) [inline Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml:14]:
```suggestion
  description: Either a list of Indicators of Compromise or an EntryID to a File containing a list.
```

File: Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+- name: compare_to_list
+  required: true
+  description: Indicators of Compromise list.
```
Comment (bakatzir) [inline Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml:17]:
```suggestion
  description: Either a list of Indicators of Compromise or an EntryID to a File containing a list.
```

**Review by bakatzir — APPROVED** [review]
very nice.
please add `isArray: true` to the args,
and generate a readme at `Packs/DeveloperTools/Scripts/CompareIndicators/README.md`

File: Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+commonfields:
+  id: CompareIndicators
+  version: -1
+name: CompareIndicators
+fromversion: 6.0.0
+script: ''
+type: python
+tags: []
+comment: Find the differences between two indicators lists.
```
Comment (yuvalbenshalom) [inline Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml:9]: just add a statement for what indicators are supported currently

**Review by yuvalbenshalom — APPROVED** [review]
very cool. see my comment

**Review by amshamah419 — APPROVED** [review]

File: Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml (hunk 4 above)
Comment (DeanArbel) [reply Packs/DeveloperTools/Scripts/CompareIndicators/CompareIndicators.yml:9]: It supports all type of indicators

**Pull request merged**